### ISSUE TITLE
Social Media Buttons In HTML And CSS

### ISSUE THREAD

**Issue by sodaney** [sodaney/sodaney.github.io#7]
This project I want everyone trying to design webpage like in the photo below.
![social media button](https://user-images.githubusercontent.com/45727493/53302905-69a23a00-3896-11e9-929a-0994bec69b54.jpg)
This project purpose is to practice with HTML and CSS. You guys can use any images as your background.
Enjoy coding.

**Comment by SahibArora** [comment]
Hi, 
I can do this one!
Thanks,

**Comment by sodaney** [comment]
@SahibArora Yes sure. Everyone should try to do it because it is for improving HTML and CSS skill.
Enjoy coding.!!!
